cloud/infoblox

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/python
+#
+# Copyright 2014 "Igor Feoktistov" <ifeoktistov@yahoo.com>
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import re
+import requests
+import json
+import os
+import urllib3
+urllib3.disable_warnings()
+ca_certs = "/etc/ssl/certs/ca-certificates.crt"  # Or wherever it lives.
+
+http = urllib3.PoolManager(
+    cert_reqs='CERT_REQUIRED', # Force certificate check.
+    ca_certs=ca_certs,         # Path to your certificate bundle.
+)
+class InfobloxNotFoundException(Exception):
+    pass
+
+class InfobloxNoIPavailableException(Exception):
+    pass
+
+class InfobloxGeneralException(Exception):
+    pass
+
+class InfobloxBadInputParameter(Exception):
+    pass
+
+class Infoblox(object):
+    """ Implements the following subset of Infoblox IPAM API via REST API
+       
+    """
+
+    def __init__(self, iba_ipaddr, iba_user, iba_password, iba_wapi_version, iba_dns_view, iba_network_view, iba_verify_ssl=True):
+        '''Class initialization method
+	:param iba_ipaddr: IBA IP address of management interface
+	:param iba_user: IBA user name
+	:param iba_password: IBA user password
+	:param iba_wapi_version: IBA WAPI version (example: 1.0)
+	:param iba_dns_view: IBA default view
+	:param iba_network_view: IBA default network view
+        :param iba_verify_ssl: IBA SSL certificate validation (example: False)
+        '''
+	self.iba_host = iba_ipaddr
+	self.iba_user = iba_user
+	self.iba_password = iba_password
+	self.iba_wapi_version = iba_wapi_version
+	self.iba_dns_view = iba_dns_view
+	self.iba_network_view = iba_network_view
+        self.iba_verify_ssl = iba_verify_ssl
+    
+    def get_next_available_ips(self,network,number='10'):
+	""" Implements IBA next_available_ip REST API call
+	Returns IP v4 address
+	:param network: network in CIDR format
+	"""
+	rest_url = 'https://' + self.iba_host + '/wapi/v' + self.iba_wapi_version + '/network?network=' + network + '&network_view=' + self.iba_network_view
+	try:
+	    r = requests.get(url=rest_url, auth=(self.iba_user, self.iba_password), verify=self.iba_verify_ssl)
```

Comment:
I think it would be better to use fetch_url, since this will take care of the certificate verification, etc. See https://github.com/ansible/ansible-modules-extras/pull/740/files for a example ( and associated PR )